configurations/batch/batch.ini.template

```diff
@@ -544,6 +544,7 @@ type												= KAsyncDropFolderWatcher
 maximumExecutionTime								= 3000
 scriptPath											= ../plugins/drop_folder/batch/DropFolderWatcher/KAsyncDropFolderWatcherExe.php
 params.tags											= *
+params.mrss.xmlPath									= @WEB_DIR@/tmp/dropFolderFiles
```

Comment:
Why don't you use the common params.tempDirectoryPath that defined in [Worker] section?